openml-h2o/src/main/java/com/feedzai/openml/h2o/params/ParametersBuilderUtil.java

```diff
@@ -128,9 +141,68 @@ private ParametersBuilderUtil() { }
                 .filter(modelParam -> !"calibrate_model".equals(modelParam.getName()))
                 // We already pass a seed always whenever possible.
                 .filter(modelParam -> !"seed".equals(modelParam.getName()))
```

Comment:
Do we still need this?

Reply:
I'm going to check manually if the number of fields without those filters is the same in every case. If so, I'm going to remove it.

Reply:
They are doing some work:

DeepLearningParametersV3: all filters: 57	one filter: 58
DRFParametersV3: all filters: 19	one filter: 22
GBMParametersV3: all filters: 22	one filter: 25
NaiveBayesParametersV3: all filters: 10	one filter: 11
XGBoostParametersV3: all filters: 36	one filter: 37
GLMParametersV3: all filters: 26	one filter: 27
IsolationForestParametersV3: all filters: 10	one filter: 11

So I'm keeping them.

Reply:
The numbers are the number of returned parameters after filtering.